tdrs-frontend/src/assets/uswds/styles.scss

```diff
@@ -0,0 +1,73 @@
+/*
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+* * * * * ==============================
+========================================
+========================================
+========================================
+*/
+
+// -------------------------------------
+// Import individual theme settings
+
+@import "uswds-theme-general";
```

Comment:
Since you're importing these directly into the index.scss a level up, do you need this file?

Reply:
Nope, thank you. I removed this.